Tests/MessagingInApp/MessagingInAppImplementationTest.swift

```diff
@@ -167,4 +167,25 @@ class MessagingInAppImplementationTest: UnitTest {
         messagingInApp.action(message: givenGistMessage, currentRoute: .random, action: .random, name: .random)
         XCTAssertEqual(eventListenerMock.messageActionTakenCallsCount, 2)
     }
+
+    func test_eventListeners_expectActionListenerForCloseAction() {
```

Comment:
```suggestion
    func test_eventListeners_givenCloseAction_expectListenerEvent() {
```

I think this explains the test a little better.